lte/gateway/c/core/oai/common/common_types.h

```diff
@@ -177,9 +177,10 @@ typedef enum {
   RAT_HRPD           = 2001,
   RAT_UMB            = 2002,
   RAT_EHRPD          = 2003,
+  RAT_NR             = 2004,
```

Comment:
why 2004?

Reply:
From ETSI 3GPP TS 29.212:
```
5.3.31 RAT-Type AVP
The RAT-Type AVP (AVP code 1032) is of type Enumerated and is used to identify the radio access technology that is
serving the UE. It may be further detailed by AN-Trusted if applicable.
NOTE 1: Values 0-999 are used for generic radio access technologies that can apply to different IP-CAN types and
are not IP-CAN specific.
NOTE 2: Values 1000-1999 are used for 3GPP specific radio access technology types.
NOTE 3: Values 2000-2999 are used for 3GPP2 specific radio access technology types.
NOTE 4: The informative Annex C presents a mapping between the code values for different access network types.
The following values are defined:
WLAN (0) 
 This value shall be used to indicate that the RAT is WLAN.
VIRTUAL (1)
 This value shall be used to indicate that the RAT is unknown. For further details refer to 3GPP TS 29.274 [22].
UTRAN (1000)
 This value shall be used to indicate that the RAT is UTRAN. For further details refer to 3GPP TS 29.060 [18].
GERAN (1001)
 This value shall be used to indicate that the RAT is GERAN. For further details refer to 3GPP TS 29.060 [18].
GAN (1002)
 This value shall be used to indicate that the RAT is GAN. For further details refer to 3GPP TS 29.060 [18] and
3GPP TS 43.318 [29].
HSPA_EVOLUTION (1003)
 This value shall be used to indicate that the RAT is HSPA Evolution. For further details refer to
3GPP TS 29.060 [18].
EUTRAN (1004)
 This value shall be used to indicate that the RAT is EUTRAN (WB-EUTRAN). For further details refer to
3GPP TS 29.274 [22].
EUTRAN-NB-IoT (1005)
 This value shall be used to indicate that the RAT is NB-IoT. For further details refer to 3GPP TS 29.274 [22].
NG-RAN (1006)
 This value shall be used to indicate that the RAT is NG-RAN.
NOTE 5: This value is not used in the present specification.
CDMA2000_1X (2000)
 This value shall be used to indicate that the RAT is CDMA2000 1X. For further details refer to
3GPP2 X.S0011 [20].
HRPD (2001)
 This value shall be used to indicate that the RAT is HRPD. For further details refer to 3GPP2 X.S0011 [20].
UMB (2002)
 This value shall be used to indicate that the RAT is UMB. For further details refer to 3GPP2 X.S0011 [20].
EHRPD (2003)
 This value shall be used to indicate that the RAT is eHRPD. For further details refer to 3GPP2 X.S0057 [24]. 
```

Reply:
Thanks for the RAN type details, updated the same